doc/conf.py

```diff
@@ -33,6 +34,7 @@
 os.environ["PYOPENCL_TEST"] = "port:cpu"
 
 nitpick_ignore_regex = [
+    ["py:mod", r"jax"],  # FIXME: not sure why this does not work
```

Comment:
That object doesn't seem to be in their `objects.inv` (from a quick search). Maybe use `jax.numpy` instead? Or just a link `jax <https://docs.jax.dev>`.